msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/mdm/TestCase833526.java

```diff
@@ -49,6 +50,7 @@
 // [Joined][MSAL] Device Admin MDM: Broker Auth for MDM account
 // https://identitydivision.visualstudio.com/DevEx/_workitems/edit/833526
 @SupportedBrokers(brokers = BrokerCompanyPortal.class)
+@RetryOnFailure(retryCount = 2)
```

Comment:
Can we somehow track on the pipeline how often specific tests are being retried? Probably not but this would be good thing to have in the future.

Reply:
Don't think so, we might be able to log every time a retry happens though.

Reply:
Looks like this is already being logged in RetryTestRule

Reply:
We probably need telemetry for tests and test infra as well. (We've done this in the past actually for a different set of non-UI tests). Now that we have actually a proper telemetry stream, we can discuss taking that further in the future. I think with our own Dashboards we might be able to better track the reliability and go down to specific errors reoccurring in specific test cases. All this is for the future thought - just leaving some thoughts :)